packages/shared/components/filters/items/SelectionFilterItem.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="typescript">
+    import type { SelectionFilterUnit } from '@core/wallet/interfaces'
+    import { Dropdown } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import type { DropdownChoice } from '@core/utils'
+
+    export let filterUnit: SelectionFilterUnit
+
+    const choices: DropdownChoice[] = filterUnit.choices.map((choice) => ({
+        label: localize(`${filterUnit.localeKey}.${choice}`),
+        value: choice,
+    }))
+
+    $: value = localize(`${filterUnit.localeKey}.${filterUnit.selected}`)
+
+    function onSelect(item): void {
```

Comment:
I think that readability would be improved by having the typing for the param here

Reply:
Tried to do type it with `DropdownChoice` but then the next line: 
`filterUnit.selected = item.value` 
fails because `item.value` is `unknown`.  If i do `filterUnit.selected = item.value as string` (or what the corresponding other type in the other similar files`, something in our precommit deletes this part and doesn't let me push it